Frontend/ui-driver/src/Screens/RegistrationScreen/View.purs

```diff
@@ -372,12 +375,12 @@ listItem push item state =
     , orientation HORIZONTAL
     , padding $ Padding 12 12 12 12
     , cornerRadius 8.0
-    , visibility $ boolToVisibility $ compVisibility state item
+    , visibility $ boolToVisibility $ (compVisibility state item && ( if item.stage == ST.PAN_CARD || item.stage == ST.AADHAAR_CARD || item.stage == ST.PROFILE_PHOTO then state.data.config.showProfileAadhaarPan else true))
```

Comment:
`if (elem item.stage [ST.PAN_CARD, ST.AADHAAR_CARD, ST.PROFILE_PHOTO]) then state.data.config.showProfileAadhaarPlan else true`

Reply:
Also this `showProfileAadhaarPan` is not required because if `AADHAAR_CARD` or any other item is coming in compulsory then due to frontend config the driver won't be able to see the option and not able to onboard because its compulsory.